src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -404,6 +404,9 @@ export class PatientViewPageStore {
                 return [];
             }
 
+        },
+        onResult: (result:DiscreteCopyNumberData[])=>{

```

Comment:
this will accumulate across patients, yes?  that is, samples from multiple patients.  is that ok?